packages/account-abstraction/src/BaseAccountAPI.ts

```diff
@@ -144,6 +146,10 @@ export abstract class BaseAccountAPI {
     isDelegateCall: boolean
   ): Promise<string>
 
+  abstract encodeExecuteCall(target: string, value: BigNumberish, data: BytesLike): Promise<string>
```

Comment:
should we still keep one abstract method encodeExecute() and in SmartAccountAPI implement it to use encodeExcuteCall()